int64 A
---
int64 joint_angle_1
int64 joint_angle_2
